frontend-nextjs/vitest.config.ts

```
@@ -2,11 +2,12 @@
 
 import { defineConfig } from 'vitest/config'
 import react from '@vitejs/plugin-react'
- 
+import path from 'path';
```

Biome

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)